### PR TITLE
mrchanmp.ngo.us

### PR DESCRIPTION
**Pull request by mrchanmp**
# Pull Request Template for `.ngo.us` Domain Registration

### 1. Zone File Edit

- [x] **Alphabetical Order**: Confirm your domain block is added in alphabetical order by your organization’s name.
- [x] **Two Comment Lines**: 
  - First line: Full organization name.
  - Second line: Organizational contact email (must be different from the `.ngo.us` domain applied for).
- [x] **Nameserver Entries**:
  - At least two NS (Nameserver) records with a TTL value of `86400`.
  - Correct format for the domain block (see README for examples).

Example:
```
; Happy Cat Foundation
; Submitted by networking team <happycat@example.org>
happycat 86400 IN NS ns1.example.com.
happycat 86400 IN NS ns2.example.com.
```

---

#### c. **Proof of Domain Mention**

---

### 3. Agreement to Terms

By submitting this pull request, I agree to the following:
- My organization is a non-commercial entity that is not affiliated with any government body.
- My organization operates legally and in good faith according to the stated purpose.
- I have provided accurate and truthful information throughout this application.
- Other terms outlined in the Registry's [Terms of Service](https://nic.ngo.us/terms-of-service/).

---

### 4. Final Checklist Before Submitting

- [x] Zone file edited correctly, following format and alphabetical order.
- [x] At least two valid nameservers with correct TTL (86400).
- [x] Domain block includes required comments with organization name and contact email.
- [x] Intended use of the domain clearly explained.
- [x] URLs for verification included with the exact domain name mentioned on the public profile.
- [x] Agreement to terms signed.

---

### CODE REVIEW

**Comment by mrchanmp** [comment]
I Agree Terms of Service

**Review by mrchanmp — COMMENTED** [review]
done

**Comment by mrchanmp** [comment]
; MrChanMP Homeweb
; Submitted by Admin Team <cmp@tutanota.de>
mrchanmp    86400 IN  NS  ns1.hstn.me.
mrchanmp    86400 IN  NS  ns2.hstn.me.

File: db.ngo.us [hunk 1]
```diff
@@ -33,6 +33,11 @@ _whois._tcp	3600	IN	SRV	0 0 43 whois.ngo.us.
 
 ; ========== BEGIN ORGANIZATIONS ==============
 
+; MrChanMP Homeweb
```
Comment (ngoregistry) [inline db.ngo.us:36]: Please read the requirements carefully. Your domain block is added in alphabetical order by your organization’s name.

**Comment by ngoregistry** [comment]
Please fill out the PR Description. DO NOT LEAVE IT EMPTY.

**Pull request closed without merging**